src/handlers/transfers/handler.js

```diff
@@ -120,8 +112,9 @@ const prepare = async (error, messages) => {
       histTimerEnd({ success: false, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId })
       return true
     }
-    // Action Letter (AL) is used to denote breadcrumb flow end
-    const AL = action === TEA.PREPARE ? 'P' : '?'
+    const PrepareActionLetter = 'P'
```

Comment:
Should all of these not be put in an enum in the enum file?

Reply:
yes, it makes sense